h2o-docs/src/product/data-science/algo-params/alpha.rst

```diff
@@ -0,0 +1,146 @@
+
+ :math:`P(\alpha,\beta) = (1 - \alpha) /2 ||\beta||{^2_2} + \alpha||\beta||_1 = \sum_j[(1 - \alpha) /2\beta{^2_j} + \alpha|\beta_j|]`
+
+Given the above, a value of 1.0 represents LASSO, and a value of 0.0 produces ridge regression. This value defaults to 0.5.
+
+This option also works closely with the `lambda <lambda.html>`__ parameter, which controls the amount of regularization applied. The following table describes the type of penalized model that results based on the values specifed for the ``lambda`` and ``alpha`` options.
+
++------------------+-----------------------+------------------------------------------+
+| ``lambda`` value | ``alpha`` value       | Result                                   |
++==================+=======================+==========================================+
+| ``lambda`` == 0  | ``alpha`` = any value | No regularization. ``alpha`` is ignored. |
++------------------+-----------------------+------------------------------------------+
+| ``lambda`` > 0   | ``alpha`` == 0        | Ridge Regression                         |
++------------------+-----------------------+------------------------------------------+
+| ``lambda`` > 0   | ``alpha`` == 1        | LASSO                                    |
++------------------+-----------------------+------------------------------------------+
+| ``lambda`` > 0   | 0 < ``alpha`` < 1     | Elastic Net Penalty                      |
++------------------+-----------------------+------------------------------------------+
+
+Related Parameters
+~~~~~~~~~~~~~~~~~~
+
+- `lambda <lambda.html>`__
+
+
+Example
+~~~~~~~
+
+.. example-code::
+   .. code-block:: r
+
+	library(h2o)
+	h2o.init()
+
+	# import the boston dataset:
+	# this dataset looks at features of the boston suburbs and predicts median housing prices
+	# the original dataset can be found at https://archive.ics.uci.edu/ml/datasets/Housing
+	boston <- h2o.importFile("https://s3.amazonaws.com/h2o-public-test-data/smalldata/gbm_test/BostonHousing.csv")
+
+	# set the predictor names and the response column name
+	predictors <- colnames(boston)[1:13]
+	# set the response column to "medv", the median value of owner-occupied homes in $1000's
+	response <- "medv"
+
+	# convert the chas column to a factor (chas = Charles River dummy variable (= 1 if tract bounds river; 0 otherwise))
+	boston["chas"] <- as.factor(boston["chas"])
+
+	# split into train and validation sets
+	boston.splits <- h2o.splitFrame(data =  boston, ratios = .8, seed = 1234)
+	train <- boston.splits[[1]]
+	valid <- boston.splits[[2]]
+
+	# try using the `alpha` parameter:
+	# train your model, where you specify alpha
+	boston_glm <- h2o.glm(x = predictors, y = response, training_frame = train,
+	                      validation_frame = valid,
+	                      alpha = .25,
+	                      seed = 1234)
```

Comment:
technically no need for the seed here. Minor detail but it might suggest there is some randomness in the algo.